keen/explorer
# Fixes #81

## What does this PR do? How does it affect users?
Ensures group_by property of queries is encoded into JSON before being uri encoded - as per the Keen API. We were not doing this before because we were not supporting multi-group by - but now that we are, we must do this.

## How should this be tested?

1) Try creating single and multi-group by queries.
2) Run them.
3) Copy and paste the API Query URL into a new tab.
4) Ensure the response JSON looks as expected.

Step through the code line by line. Things to keep in mind as you review:
 - Are there any edge cases not covered by this code?
 - Does this code follow conventions (naming, formatting, modularization, etc) where applicable?

Fetch the branch and/or deploy to staging to test the following:


## Related tickets?
#81
